pupil-spa/src/assets/shared-styles/elements/_check.scss

```diff
@@ -1,70 +1,127 @@
 // Questions
+
+$vertical-breakpoint-1: 600px;
```

Comment:
@jon-shipley are we supposed to be using ems as per gds guidance?
If so we might consider changing these with pem()?

Reply:
@GeorgeChatzigiannis - we should be using `pem()` to convert px to em, however, I'm not touching this design again now that it's working.

We will make minor changes after the June trial - that's your chance for these comments.